docker/generic/start_proxy.py

```diff
@@ -182,6 +182,13 @@ def make_argparser():
         Cipher suites to use for downstream connections as a comma-separated list.
         Please refer to https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/auth/common.proto#auth-tlsparameters''')
 
+    parser.add_argument('--ssl_server_root_cert_path', default=None, help='''
```

Comment:
mTLS means:
* verify client certificates
* specify which clients are allowed.  Do we want to support that? or it is in a separate pr?

Reply:
You are right. This PR, like our existing TLS connections, only do the basic server certs validation/authn. As for tls access control, envoy does have a bunch of other [options](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto#extensions-transport-sockets-tls-v3-certificatevalidationcontext) to do access control or extra validation. I am thinking we make another ones if necessary after asking them what they need. We can even encourage to contribute based on this PR.